Tickr/TickrTests/TickrTestData.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  TickrTestData.swift
+//  Tickr
+//
+//  Created by Stephen Payne on 4/10/16.
+//  Copyright © 2016 Stephen Payne. All rights reserved.
+//
+
+import Foundation
+@testable import Tickr
+
+//Will create test data for unit tests upon instantiation
+class TickrTestData {
```

Comment:
because of the `TickrTestData` all of your tests are correlated and not independent